### PR TITLE
Feature/weos 1289 - As a developer I should be able to specify that an endpoint request body is form data

### PR DESCRIPTION
**Pull request by RandyDeo**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by Aidan-E — DISMISSED** [review]

File: controllers/rest/controller_standard.go [hunk 1]
```diff
@@ -46,7 +46,36 @@ func (c *StandardControllers) Create(app model.Service, spec *openapi3.Swagger,
 			})
 		}
 		//reads the request body
-		payload, _ := ioutil.ReadAll(ctxt.Request().Body)
+		//payload, _ := ioutil.ReadAll(ctxt.Request().Body)
```
Comment (akeemphilbert) [inline controllers/rest/controller_standard.go]: if this is no longer needed please remove

File: features/create-content.feature [hunk 2]
```diff
@@ -182,15 +197,26 @@ Feature: Create content
...
       Then the "Post" is created
         | title          | description                       |
-        | Some Blog      | Some Description                  |
+        | Some Post      | Some Description                  |
       And the "Post" should have an id
       And the "ETag" header should be present
 
-    @skipped
-    @WEOS-1289 
+    @WEOS-1289 @skipped
```
Comment (akeemphilbert) [inline features/create-content.feature]: this is still being skipped

File: features/create-content.feature [hunk 3]
```diff
@@ -182,15 +197,26 @@ Feature: Create content
...
+      Given "Sojourner" is on the "Category" create screen
+      And "Sojourner" enters "Some Category" in the "title" field
+      When the "Category" form is submitted with content type "multipart/form-data"
+      Then the "Category" is created
+        | title          |
+        | Some Category  |
+      And the "Category" should have an id
+      And the "ETag" header should be present
+
+    @WEOS-1289 @skipped
```
Comment (akeemphilbert) [inline features/create-content.feature]: this is still being skipped

**Review by akeemphilbert — CHANGES_REQUESTED** [review]
There are few relevant scenarios that are skipped

**Review by akeemphilbert — APPROVED** [review]

**Pull request merged**